puente/ext.py

```diff
@@ -5,10 +5,10 @@
 
 
 @jinja2.contextfunction
-def _gettext_alias(context, text, *args, **kwargs):
+def _gettext_alias(context, *args, **kwargs):
```

Comment:
Would this be passing in the context or the text? And why is context so negative against the text anyway, why can't he just be protext?